helix-term/Cargo.toml

```diff
@@ -55,5 +56,7 @@ toml = "0.5"
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 
+smallvec = "1.4"
```

Comment:
helix_core re-exports `SmallVec` so you can use that, this way we don't have to specify the version twice